packages/vulcan-lib/lib/modules/components.js

```diff
@@ -112,3 +113,27 @@ export const populateComponentsApp = () => {
 export const copyHoCs = (sourceComponent, targetComponent) => {
   return compose(...sourceComponent.hocs)(targetComponent);
 }
+
+/**
+ * Returns an instance of the given component name of function
+ * @param {string|function} component  A component or registered component name
+ * @param {Object} [props]  Optional properties to pass to the component
+ */
+//eslint-disable-next-line react/display-name
+export const instantiateComponent = (component, props) => {
```

Comment:
What is this for? If it's to let people pass either a component or a component name, it think it's better to just accept two different props (e.g. `afterComponent` and `afterComponentName`); or just force it to be a string. I'm not sure the extra magic is a good idea.

Reply:
OK, let's keep it then.